src/components/Header.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import "../css/App.css";
+
+function Header(){
+  return(
+    <div className="header">

```

Comment:
No need to create it as a reusable component as  I think it won't be reused again right?